code/datums/martial/nemesis_kick.dm

```diff
@@ -0,0 +1,79 @@
+#define NEMESIS_MAX_CHARGE		 	20
+#define NEMESIS_CHARGE_PER_MINE		 3
+#define NEMESIS_CHARGE_PER_JUMP		 2
+#define NEMESIS_CHARGE_PER_KNOCKDOWN 2
+#define NEMESIS_CHARGE_PER_PARALYZE  6
+
+#define NEMESIS_KNOCKDOWN_LENGTH 	 2 SECONDS
+#define NEMESIS_PARALYZIS_LENGTH	 5 SECONDS
+
+/datum/martial_art/nemesis //Funny, but there are no actuall kicks in this bootleg martial art
+	name = "Nemesis Kick"
+	id = MARTIALART_NEMESIS
+	block_chance = 15 //Lets give them a small block chance, Nemesis Solutions equipment is also based around defencive moves
```

Comment:
you do know that this block_chance requires you to have throw mode on for it to work, right? since the successful block chance is only 15%, this is almost never going to come up.

also, this is pretty unnecessary in general.